assets/src/application/ui/forms/ee-form/Submit.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { useForm } from 'react-final-form';
+
+import { Button } from './adapters';
+import { SubmitProps } from './types';
+
+const Submit: React.FC<SubmitProps> = ({ submitting, submitButton, resetButton }) => {
+	const form = useForm();
+	return (
+		<div className='submit-wrapper'>
+			<Button
+				icon='save'
+				htmlType='submit'
+				disabled={submitting}
+				loading={submitting}
+				className='submit-button'
+				type='primary'
+				{...submitButton}
+			>
+				{submitButton.label || __('Submit')}
+			</Button>
+			&nbsp;
```

Comment:
`{ " " }`

Reply:
That is messed up by prettier and `&nbsp;` is perfectly supported by JSX.